src/main/javascript/components/datepicker/create-datepicker.js

```diff
@@ -36,6 +36,8 @@ const datepickerClassnames = {
 export async function createDatepicker(selector, { urlPrefix, getPersonId, onSelect = noop }) {
   const { localisation } = window.uv.datepicker;
 
+  /* TODO add el as new language*/
```

Comment:
should I create new variables on that section??

Reply:
Good morning @nikoszagk, I will take a look at it and prepare something. That should not be much and then you can make a code review of the change so for the next language you can help he people! :D

Reply:
I never thought I will see the UV in greek. Looks pretty cool! :) So time to learn some.

![grafik](https://user-images.githubusercontent.com/5683677/157413066-08c6d263-a93d-42ed-9b99-27a0519c8ff9.png)

Reply:
@derTobsch in [src/main/javascript/bundles/date-fns-localized.js I added the greek "el" language. with the editing of the [create-datepicker.js is ok to go online?

Reply:
I need to check again the greek language for some mistakes. Note that I am developer, not teacher to a school for greek lessons :D :D

Reply:
I think it does look good now. @nikoszagk what date format do you use in greek? Is it the same as in english oder german?

Reply:
It's like dd/mm/yyyy in Greece

Reply:
hm ok, so lets start with dd.MM.yyyy - and change it in the future if it does not fit anymore. Hope that is ok for you?